<!--   -->

<launch>
	<node name="usbcan_driver_node" pkg="usbcan_driver" type="usbcan_driver_node" output="screen">
	
	<rosparam param="acc_code" >  [0x53F,0x000] </rosparam>
	<rosparam param="mask_code" > [0x03F,0x000] </rosparam>
	
	<rosparam param="baudrate" > [500,500] </rosparam>
	<rosparam param="frame_id" > ["ch1","ch2"] </rosparam>
	
	<!--from_can_topic can分析仪发送的话题-->
	<!--to_can_topic   can分析仪接收的话题-->
	<param name ="from_can_topic"  type ="string"  value="/from_can_topic"/>
	<param name ="to_can_topic"    type ="string"  value="/to_can_topic"/>
	
	
	</node>

</launch>
